tests/testCmdLineTask.py

```diff
@@ -150,7 +150,8 @@ def testNoBackupConfig(self):
         TestTask.parseAndRun(args=[DataPath, "--output", self.outPath, "--id", "visit=3", "filter=r",
                                    "--config", "floatField=-99.9", "--clobber-config", "--no-backup-config"])
         # Ensure backup config file was NOT created
-        self.assertFalse(os.path.exists(os.path.join(self.outPath, "config", TestTask._DefaultName + ".py~1")))
+        self.assertFalse(
+            os.path.exists(os.path.join(self.outPath, "config",TestTask._DefaultName + ".py~1")))
```

Comment:
For this test? Certainly. In general, we don't enforce it, but it should usually be so.